node-zerox/src/utils/file.ts

```diff
@@ -66,6 +67,34 @@ export const downloadFile = async ({
   return { extension, localPath };
 };
 
+// Convert HEIC file to JPEG
+export const convertHeicToJpeg = async ({
+  localPath,
+  tempDir,
+}: {
+  localPath: string;
+  tempDir: string;
+}): Promise<string> => {
+  try {
+    const inputBuffer = await fs.readFile(localPath);
+    const outputBuffer = await heicConvert({
+      buffer: inputBuffer,
+      format: "JPEG",
+      quality: 1,
+    });
+
+    const jpegPath = path.join(
+      tempDir,
+      `${path.basename(localPath, ".heic")}.jpg`
+    );
+    await fs.writeFile(jpegPath, Buffer.from(outputBuffer));
+    return jpegPath;
+  } catch (err) {
+    console.error(`Error converting HEIC to JPEG:`, err);
```

Comment:
Small nit, let's update this to `.heic to .jpeg:` so that it matches the other error messages.